controller/build.gradle

```diff
@@ -11,6 +11,9 @@ dependencies {
 
   implementation "gov.va.starter:exceptions"
 
+  // https://mvnrepository.com/artifact/javax.servlet/javax.servlet-api
+  compileOnly group: 'javax.servlet', name: 'javax.servlet-api', version: '4.0.1'
+
```

Comment:
This was necessary to pull in for `org.springframework.web.HttpRequestMethodNotSupportedException` to register with the app. Otherwise, without this, it would explain about `error: cannot access ServletException`